frontend/src/components/PostAddingForm/PostAddingForm.jsx

```diff
@@ -110,11 +137,31 @@ const PostAddingForm = ({ formSubmit }) => {
       <ContentInput
         type="text"
         placeholder="Enter announcement message..."
-        {...register('content', { required: true })}
+        {...register('content', {
+          required: "This field can't be empty",
+          pattern: {
+            value: /.*[^\s].*/,
+            message: "This field can't contain only whitespaces",
+          },
+          minLength: {
+            value: 3,
+            message: 'Your message should be at least 3 characters long',
```

Comment:
Tu bym ustawił z 20/30 bo co to za ogłoszenie, które ma 3 znaki treści?  xD

Reply:
No w sumie ta XD mi to sie ogolnie to ograniczenie nie podoba bo moge znalezc duzo przykladow gdzie wiecej niz 10 znakow uzywac nie trzeba, ale jak juz jest to zrobie jak mowisz :D